src/shared/ui/date-lang/date-lang.tsx

```diff
@@ -1,31 +1,38 @@
-/* eslint-disable @stylistic/jsx-one-expression-per-line */
+import classNames from 'classnames/bind';
+import { COURSE_STALE_AFTER_DAYS } from '@/app/const';
+import { dayJS } from '@/app/services/dayjs';
 import micIcon from '@/shared/assets/icons/mic.svg';
 import noteIcon from '@/shared/assets/icons/note-icon.svg';
+import { getCourseDate } from '@/shared/helpers/getCourseDate';
 import Image from '@/shared/ui/image';
 
 import styles from './date-lang.module.scss';
 
+const cx = classNames.bind(styles);
+
 interface DateLangProps {
   startDate: string;
   mode: string;
   language: string[];
```

Comment:
Why Array? Maybe it would be better to use 'string'? We only use one language at a time

Reply:
It would be better, but right know too much code tied with this array type and i don't see any benefits in changing it